packages/CollabCodeEdit-Client.package/CCESession.class/instance/joinSessionWith.and..st

```diff
@@ -0,0 +1,5 @@
+session-managment
+joinSessionWith: aSessionScopeArray and: aHash
+		((CCEPackageHasher hashPackages: aSessionScopeArray) = aHash)
+			ifTrue: [ self joinCleanSessionWith: aSessionScopeArray and: aHash ]
+			ifFalse: [	 self joinDirtySessionWith: aSessionScopeArray and: aHash ].
```

Comment:
Remove additional white space in `ifFalse:` block

Reply:
Must be another git issue...